docker-compose.yml

```diff
@@ -12,7 +12,7 @@ services:
     volumes:
       - 'flask-app-db:/var/lib/postgresql/data'
     ports:
-      - '5433:5432'
+      - '5434:5432'
```

Comment:
Czemu port zmieniony w docker-compose?

Reply:
prosze to wycofać

Reply:
Port miałem zajęty, gdy wszystkie błędy będą poprawione przywrócę na 5433

Reply:
Gdy jest taka sytuacja to nie pushuj takiej zmiany na PR.